spec/mongoid_search_spec.rb

```diff
@@ -280,7 +289,8 @@
   context "relevant search" do
     before do
       Mongoid::Search.relevant_search = true
-      @imac = Product.create :name => 'apple imac'
+      category = Category.create(name: 'New Category')
+      @imac = Product.create :name => 'apple imac', category: category 
```

Comment:
Why have you added category here? If there is an error if an object without category is created, we should change validations for Product class